server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaToResponseBody.scala

```diff
@@ -55,7 +55,12 @@ private[akkahttp] class AkkaToResponseBody(implicit ec: ExecutionContext, m: Mat
       case RawBodyType.ByteArrayBody   => HttpEntity(ct, r)
       case RawBodyType.ByteBufferBody  => HttpEntity(ct, ByteString(r))
       case RawBodyType.InputStreamBody => streamToEntity(ct, contentLength, StreamConverters.fromInputStream(() => r))
-      case RawBodyType.FileBody        => HttpEntity.fromPath(ct, r.toPath)
+      case RawBodyType.FileBody        =>
+        val file = r.asInstanceOf[FileRange]
+        FileChunk.prepare(file)
```

Comment:
with akka-http, chances are there's a way to obtaina non-blocking stream (so instead of an `InputStream` converted to a `Source`, creating the `Source` directly). If possible, we should use that here